src/keys.rs

```diff
@@ -36,6 +36,7 @@ pub enum KeyPress {
     Up,
 }
 
+#[cfg_attr(not(any(unix, windows)), allow(dead_code))]
```

Comment:
`#[cfg(not(target_arch = "wasm32"))]` ?

Reply:
instead of attaching the allow dead code annotation, I've tagged this function with `cfg(any(windows, unix))` since this function is only used in these backends.